### DISCUSSION TITLE
Problem with some css using the * notation in Angular

### DISCUSSION THREAD

**Discussion by mbouclas** [GrapesJS/grapesjs#3925] [Q&A]
Been using grapes with Angular for the past 2 years, but since the latest version of Angular (v13) i'm getting some CSS warnings on styles like
```
.sp-palette{*width:220px;max-width:220px}
```
or
```
.sp-container{position:absolute;top:0;left:0;display:inline-block;*display:inline;*zoom:1
```
The warning is `warning: Expected identifier but found "*"`. Could be that Angular dropped support for IE11.
Anyone else had this?

**Comment by ronaldohoch** [comment]
What's the point of putting the * in the css rule, I've never seen it in css or Angular.

**Reply by jmtt89** [reply]
is a hack for IE7 and before
"*property: value applies the property value in IE 7 and below. It may or may not work in future versions. Warning: this uses invalid CSS."

**Comment by mbouclas** [comment]
I'm not sure why it's used exactly, but it's in `grapes.min.css` all over the place and the Angular compiler doesn't seem to like it

**Comment by artf** [comment]
That seems to come from the spectrum-colorpicker. I'll try to see what I can do.

**Reply by jmtt89** [reply]
umm i guess we're out of luck

https://github.com/bgrins/spectrum/issues/205
https://github.com/bgrins/spectrum/pull/203
https://github.com/bgrins/spectrum/issues/426

**Reply by artf** [reply]
Not a problem, I included now the CSS in grapesjs source (JS was already there with its own fixes). In long-term, all UI stuff should be outside of the core.
